packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/trade-type-tab.jsx

```diff
@@ -0,0 +1,50 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ButtonToggle } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const TradeTypeTab = ({ className, onChange }) => {
+    const [selectedValue, setSelectedValue] = React.useState('');
+    const tab_list = [{ text: localize('Long'), value: 'turboslong' }];
+
+    const has_short = tab_list.find(tab => tab.value === 'turbosshort');
```

Comment:
You create `tab_list` array above (line 10), so you know, that it doesn't contain `turbosshort`. Why do you need to check it (line 12) and than add (13-15)?

Reply:
Noted on this thing. I try to replicate the advanced_duration_component, that would be the reason of this.
Reverting back to initial array value.